### PR TITLE
[SYCLomatic] Remove custom helper file related LIT cases and 2 options

### PR DESCRIPTION
**Pull request by zhiweij1**
Signed-off-by: Jiang, Zhiwei <zhiwei.jiang@intel.com>

### CODE REVIEW

**Review by tomflinda — APPROVED** [review]
LGTM

**Review by ziranzha — APPROVED** [review]

File: clang/lib/DPCT/Error.h [hunk 1]
```diff
@@ -46,15 +46,11 @@ enum ProcessStatus {
...
   MigrationErrorInvalidExplicitNamespace = -33,
-  MigrationErrorCustomHelperFileNameContainInvalidChar = -34,
-  MigrationErrorCustomHelperFileNameTooLong = -35,
-  MigrationErrorCustomHelperFileNamePathTooLong = -36,
   MigrationErrorDifferentOptSet = -37,
   MigrationErrorInvalidRuleFilePath = -38,
   MigrationErrorCannotParseRuleFile = -39,
   MigrationErrorInvalidAnalysisScope = -40,
   MigrationErrorInvalidChangeFilenameExtension = -41,
-  MigrationErrorConflictOptions = -42,
```
Comment (zhimingwang36) [inline clang/lib/DPCT/Error.h]: let's keep MigrationErrorConflictOptions

**Review by zhimingwang36 — APPROVED** [review]
LGTM

File: clang/lib/DPCT/Error.h (hunk 1 above)
Comment (zhiweij1) [reply clang/lib/DPCT/Error.h]: Done

**Pull request merged**